core/Util/PlasmaVersions.h

```diff
@@ -85,6 +85,7 @@ class HSPLASMA_EXPORT PlasmaVer
         pvPrime     = MAKE_VERSION(2, 0, 63, 11),
         pvPots      = MAKE_VERSION(2, 0, 63, 12),
         pvMoul      = MAKE_VERSION(2, 0, 70,  0),
+        pvMqo       = MAKE_VERSION(2, 0, 70,  1),
```

Comment:
H-uru/Plasma is currently 2.0.70.2, FWIW.

Reply:
That ends up not really mattering because that's handled via the ClassVersions stuff.

MQO isn't really 2.0.70.1, we just need something to use as a flag because of the missing ClassVersions for messages :(